History/js/new.js

```diff
@@ -235,7 +235,7 @@ var callbacks = {
       iFrame.setAttribute("data-privly-accept-resize","true");
 
       // The href of the original link as dictated by the remote server
-      var canonicalHref = $(this).attr("data-canonical-href");
+      var canonicalHref = $reference.attr("data-canonical-href");
```

Comment:
Generally the Priv.ly code base doesn't prepend variables with the `$` sign. However, since this is a jquery reference I think this could be good convention.